crates/driver/src/domain/competition/order/mod.rs

```diff
@@ -28,6 +30,7 @@ pub struct Order {
     pub kind: Kind,
     pub app_data: AppData,
     pub partial: Partial,
+    pub executed: U256,
```

Comment:
1. this information is already covered by `partial`
2. if this were to be added it should use the new types like `eth::TokenAmount`.

Reply:
Ok, being that the case, I'll remove it! Thanks.